src/Service/ValidarCpfService.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Service;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+
+class ValidarCpfService
+{
+    private HttpClientInterface $httpClient;
+    private string $token;
+    private LoggerInterface $logger;
+
+    public function __construct(HttpClientInterface $httpClient, string $token, LoggerInterface $logger)
+    {
+        $this->httpClient = $httpClient;
+        $this->token = $token;
+        $this->logger = $logger;
+    }
+
+    public function execute(string $cpf): bool
+    {
+        if (!$this->validarCpfLocalmente($cpf)) {
+            return false;
+        }
+
+        try {
+            $response = $this->httpClient->request('GET', 'https://api.invertexto.com/v1/validator', [
+                'query' => [
+                    'value' => $cpf,
+                    'type' => 'cpf',
+                    'token' => $this->token,
+                ],
+            ]);
+
+            $data = $response->toArray();
+
+            return $data['valid'] ?? false;
+
```

Comment:
só para não precisa da variável...
```suggestion
            return $response->toArray()['valid'] ?? false;

```